### ISSUE TITLE
Change pyGSM to pyGDSM

### ISSUE THREAD

**Issue by kbharatgehlot** [AlanLoh/nenupy#54]
The pyGSM dependency of nenupy is deprecated for a while. The new module is called pyGDSM.

**Comment by AlanLoh** [comment]
Indeed. However, only the Oliveira-Costa et al. 2008 model, in HEALPix, is used in nenupy and for simplicity I haven't changed the dependency.
If the newest features are required, I'll switch to pygdsm (or even just to get the most up-to-date packages, you're right). Do you have something in particular in mind?
Thanks

**Comment by kbharatgehlot** [comment]
The new pygdsm also contains the LWA1 sky survey and the upcoming sky-maps will also be implemented in pygdsm. I guess it's also wise to switch over to avoid future issues due deprecated repo.
